libraries/CurieBLE/src/internal/BLEDeviceManager.cpp

```diff
@@ -133,7 +133,8 @@ void BLEDeviceManager::poll()
 }
 
 void BLEDeviceManager::end()
```

Comment:
Good point, Erik.  Did talk to Liang about this issue.

The BLEDeviceManager class is a singleton.  It is not expected to go away.  However, it can end its operation and that's where end() can come in to tidy up its allocated resources.  This will be done in the next release as part of the BLE performance improvement.  Just create a Jira ticket #839 to track this task.